pkg/autodiscovery/listeners/docker.go

```diff
@@ -411,27 +411,29 @@ func (s *DockerService) GetPorts() ([]int, error) {
 		}
 	}
 
-	sort.Ints(ports)
+	sort.Slice(ports, func(i, j int) bool {
```

Comment:
I guess you also need to sort the slice in other listeners?

Reply:
I'll double check (and add more tests since nothing failed because of sorting issues).